juju/client/_client1.py

```diff
@@ -738,9 +738,11 @@ class CAASApplicationProvisionerFacade(Type):
                                                   'name': {'type': 'string'}},
                                    'type': 'object'},
                      'CharmContainer': {'additionalProperties': False,
-                                        'properties': {'mounts': {'items': {'$ref': '#/definitions/CharmMount'},
+                                        'properties': {'gid': {'type': 'integer'},
```

Comment:
Somehow adding client20 updated all the other clients, like client1 here.

I'm not sure what's going on, should these changes be pulled in or not?